docs/source/quickstart.md

```diff
@@ -358,34 +317,27 @@ You can also retrieve the individual vectorial impulses:
 >>> tuple(val.decompose([u.km, u.s]) for val in hoh[1])
 (<Quantity 15729.741535747102 s>, <Quantity [ 0.        , 1.41999995, 0.        ] km / s>)
 ```
-```{eval-rst}
-.. _Hohmann: https://en.wikipedia.org/wiki/Hohmann_transfer_orbit
-.. _bielliptic: https://en.wikipedia.org/wiki/Bi-elliptic_transfer
+
+* [Hohmann](https://en.wikipedia.org/wiki/Hohmann_transfer_orbit)
```

Comment:
Rather than including a list of links, these `.. _Hohmann` elements are "invisible" references that are used above. See for example https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#external-links or https://docutils.sourceforge.io/docs/user/rst/quickref.html#external-hyperlink-targets

Therefore, we should keep them "invisible". Perhaps just move these to the "There are other useful methods..." paragraph above?